test/test-helpers.h

```diff
@@ -113,6 +114,42 @@ namespace TestHelpers {
 			std::string tempdir;
 			std::string filepath;
 	};
+
+	/*
+	 * The AssertArticleFileContent opens a file where the content of an article
+	 * was previously dumped (using for example OP_SHOWURL, or OP_OPEN with an
+	 * appropriate "pager" config value ) and checks the differents lines according
+	 * to the expected values passed as parameters.
+	 */
+	inline void AssertArticleFileContent( std::string path, std::string title, std::string author, std::string date, std::string url, std::string description) {
```

Comment:
Since `inline` doesn't necessarily mean that the function will be inlined, it's better to declare arguments as `const std::string&`.